ui-v2/app/search/filters/service.js

```diff
@@ -1,14 +1,34 @@
 import { get } from '@ember/object';
+import ucfirst from 'consul-ui/utils/ucfirst';
+const find = function(obj, term) {
+  if (Array.isArray(obj)) {
+    return obj.some(function(item) {
+      return find(item, term);
+    });
+  }
+  return obj.toLowerCase().indexOf(term) !== -1;
+};
 export default function(filterable) {
   return filterable(function(item, { s = '' }) {
     const term = s.toLowerCase();
-    return (
-      get(item, 'Name')
-        .toLowerCase()
-        .indexOf(term) !== -1 ||
-      (get(item, 'Tags') || []).some(function(item) {
-        return item.toLowerCase().indexOf(term) !== -1;
-      })
-    );
+    let status;
+    switch (true) {
+      case term.indexOf('service:') === 0:
```

Comment:
For what it's worth, Ember has had `term.startsWith('service:')` for ages and `startsWith` has been [standard lib](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith) as of ES2015.

Reply:
Ah yeah, nice, thankyou! Will sort in a sec.